doc/language_guide.md

```diff
@@ -442,6 +442,55 @@ INTO my_db.my_result_table;
     - [BARON](https://www.minlp.com/baron): used to solve the non-linear programming problems.
 - The `INTO my_db.my_result_table` means that the solved result would be saved in the `my_db.my_result_table` table.
 
+## TO RUN Syntax
+
+SQLFlow provide `TO RUN` statement to execute a runnable to execute complicated
```

Comment:
`provide` -> `provides`

`TO RUN statement` -> `the TO RUN statement`

`to execute a runnable to execute ...`. There are two many words `execute`. Try to use an alternative word.

Reply:
Good catch, updated